Gems/ROS2/Code/Tests/GNSSTest.cpp

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/UnitTest/TestTypes.h>
+#include <AzTest/AzTest.h>
+
+#include <GNSS/GNSSFormatConversions.h>
+
+namespace UnitTest
+{
+
+    class GNSSTest : public AllocatorsTestFixture
```

Comment:
This is going to need to be changed to LeakDetectionFixture now to compile due to this [change](https://github.com/o3de/o3de/commit/05c122feab34b37cab4e8df3fac7be14c6f3029f) by @burelc-amzn